<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>781</width>
    <height>606</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel{
color:white;
border:0px solid white;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-101</y>
      <width>821</width>
      <height>711</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{

	background-color: rgb(50, 50, 50);
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="frame_watch_data">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>370</y>
       <width>441</width>
       <height>321</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame{
background-color: rgb(0, 0, 0);
border:3px solid white;
border-radius:20px;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_5">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>20</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
color:white;
border:0px solid white;
}</string>
      </property>
      <property name="text">
       <string>x pathyleriniz</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>50</y>
        <width>371</width>
        <height>131</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QTextEdit{
color:white;

}</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>230</y>
        <width>141</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
color:white;
border:0px solid white;
}</string>
      </property>
      <property name="text">
       <string>watch data</string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_tim">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>380</y>
       <width>311</width>
       <height>181</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame{
background-color: rgb(0, 0, 0);
border:3px solid white;
border-radius:20px;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QTextEdit" name="textEdit_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QTextEdit{
color:white;
border-radius:20px;
}</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>46</x>
        <y>10</y>
        <width>191</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
color:white;
border:0px solid white;
}</string>
      </property>
      <property name="text">
       <string>delay update time</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>120</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QTextEdit{
color:white;
border-radius:20px;
}</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>46</x>
        <y>90</y>
        <width>191</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
color:white;
border:0px solid white;
}</string>
      </property>
      <property name="text">
       <string>update count</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_statu_main">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>100</y>
       <width>781</width>
       <height>16</height>
      </rect>
     </property>
     <property name="toolTipDuration">
      <number>-1</number>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame{
background-color: rgb(0, 0, 0);
border:3px solid white;
border-radius:20px;

}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>-30</x>
        <y>-50</y>
        <width>351</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>istediniz datanin xpatini asssagiya yapıstırın daha
 fazla x path icin arasina ! koyunuz</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QFrame" name="frame_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>241</width>
        <height>171</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame{

	
	background-color: rgb(63, 63, 63);

}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>20</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{

color:white;
border:0px solid white;
}</string>
       </property>
       <property name="text">
        <string>STATUS</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{

color:white;
border:0px solid white;
}</string>
       </property>
       <property name="text">
        <string>update =</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>50</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{

color:white;
border:0px solid white;
}</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>100</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{

color:white;
border:0px solid white;
}</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{

color:white;
border:0px solid white;
}</string>
       </property>
       <property name="text">
        <string>lates time =</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="frame_3">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>0</y>
        <width>341</width>
        <height>581</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame{

	
	background-color: rgb(63, 63, 63);

}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>20</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{

color:white;
border:0px solid white;
}</string>
       </property>
       <property name="text">
        <string>CONSOLE</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>321</width>
         <height>511</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{

color:white;
border:0px solid white;
}</string>
       </property>
       <property name="text">
        <string>LOG</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>560</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>data watcher run state=</string>
      </property>
     </widget>
     <widget class="QLabel" name="state">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>560</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>false</string>
      </property>
     </widget>
     <zorder>label_8</zorder>
     <zorder>frame_3</zorder>
     <zorder>label_3</zorder>
     <zorder>state</zorder>
     <zorder>frame_2</zorder>
    </widget>
    <widget class="QFrame" name="frame_get_Link">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>421</width>
       <height>241</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame{
background-color: rgb(0, 0, 0);
border:3px solid white;
border-radius:20px;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>180</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>get link</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>-30</x>
        <y>-50</y>
        <width>351</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>istediniz datanin xpatini asssagiya yapıstırın daha
 fazla x path icin arasina ! koyunuz</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>50</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
color:white;
border:0px solid white;
}</string>
      </property>
      <property name="text">
       <string>web site link</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>401</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QTextEdit{
color:white;

}</string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_save">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>120</y>
       <width>311</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame{
background-color: rgb(0, 0, 0);
border:3px solid white;
border-radius:20px;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>80</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
color:white;
border:0px solid white;
}</string>
      </property>
      <property name="text">
       <string>save file type</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QCheckBox" name="save_txt">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>70</width>
        <height>17</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QCheckBox{
color:white;

}</string>
      </property>
      <property name="text">
       <string>.txt</string>
      </property>
      <property name="tristate">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="save_json">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>130</y>
        <width>70</width>
        <height>17</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QCheckBox{
color:white;

}</string>
      </property>
      <property name="text">
       <string>json</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="save_data">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>30</y>
        <width>70</width>
        <height>17</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QCheckBox{
color:white;

}</string>
      </property>
      <property name="text">
       <string>save data</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>620</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color: rgb(0, 0, 0);
color:white;
}</string>
     </property>
     <property name="text">
      <string>last settings</string>
     </property>
    </widget>
    <zorder>pushButton_3</zorder>
    <zorder>frame_watch_data</zorder>
    <zorder>frame_tim</zorder>
    <zorder>frame_get_Link</zorder>
    <zorder>frame_save</zorder>
    <zorder>frame_statu_main</zorder>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
